theme/theme.inc

```diff
@@ -32,6 +34,12 @@ function islandora_internet_archive_bookreader_preprocess_islandora_internet_arc
           'token' => $token,
         ),
       )),
+      'backupUri' => url("islandora/object/{$page_pid}/datastream/{$backup_dsid}/view", array(
+        'absolute' => TRUE,
+        'query' => array(
+          'token' => $backup_token,
```

Comment:
Should that callback stay in this module or go somewhere higher up (e.g. the Islandora module), since it's literally just generating a token for any datastream's view page?